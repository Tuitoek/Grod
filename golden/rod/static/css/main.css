body{
	background-color: hsl(0, 0%, 98%);
}

h1,h2,h3,h4,h5,h6{
	color:hsl(0, 0%, 30%);
}

.box-element{
	box-shadow:hsl(0, 0%, 80%) 0 0 16px;
	background-color: #fff;
	border-radius: 4px;
	padding: 10px;
}

.thumbnail{
	width: 100%;
	height: 200px;
	-webkit-box-shadow: -1px -3px 5px -2px rgba(214,214,214,1);
    -moz-box-shadow: -1px -3px 5px -2px rgba(214,214,214,1);
    box-shadow: -1px -3px 5px -2px rgba(214,214,214,1);
}

.product{
	border-radius: 0 0 4px 4px;
}

.bg-dark{
	background-color: #4f868c!important;
}

#cart-icon{
	width:25px;
	display: inline-block;
	margin-left: 15px;
}

#cart-total{
	display: block;
	text-align: center;
	color:#fff;
	background-color: red;
	width: 20px;
	height: 25px;
	border-radius: 50%;
	font-size: 14px;
}

.col-lg-4, .col-lg-6, .col-lg-8, .col-lg-12{
	margin-top: 10px;
}

.btn{
	border-radius: 0;
}

.row-image{
	width: 100px;
}

.form-field{
	width:250px;
	display: inline-block;
	padding: 5px;
}

.cart-row{
	display: flex;
    align-items: flex-stretch;
    padding-bottom: 10px;
    margin-bottom: 10px;
    border-bottom: 1px solid #ececec;

}

.quantity{
	display: inline-block;
	font-weight: 700;
	padding-right:10px;
	

}

.chg-quantity{
	width: 12px;
	cursor: pointer;
	display: block;
	margin-top: 5px;
	transition:.1s;
}

.chg-quantity:hover{
	opacity: .6;
}


.hidden{
	display: none!important;
}

/* The sidebar menu */

.sidebar {
	height: 100%;
	/* 100% Full-height */
	width: 0;
	/* 0 width - change this with JavaScript */
	position: fixed;
	/* Stay in place */
	z-index: 1;
	/* Stay on top */
	top: 0;
	left: 0;
	background-color: #111;
	/* Black*/
	overflow-x: hidden;
	/* Disable horizontal scroll */
	padding-top: 60px;
	/* Place content 60px from the top */
	transition: 0.5s;
	/* 0.5 second transition effect to slide in the sidebar */
  }
  
<<<<<<< HEAD
}
#About .Atitle{
  font-size: 3rem;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
  font-weight: 900;
  margin-left: 3rem;
}


/* hover========================== */
.hovereffect {
  width: 100%;
  height: 50%;
  float: left;
  overflow: hidden;
  position: relative;
  text-align: center;
  cursor:default;
  background: #a3962e;
}

.hovereffect .overlay {
  width: 100%;
  height: 100%;
  position: absolute;
  overflow: hidden;
  top: 0;
  left: 0;
  padding: 50px 20px;
}




.hovereffect h2 {
  text-transform: uppercase;
  color: #fff;
  text-align: center;
  position: relative;
  font-size: 17px;
  overflow: hidden;
  padding: 0.5em 0;
  background-color: transparent;
}

.hovereffect h2:after {
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%;
  height: 2px;
  background: #fff;
  content: '';
  -webkit-transition: -webkit-transform 0.35s;
  transition: transform 0.35s;
  -webkit-transform: translate3d(-100%,0,0);
  transform: translate3d(-100%,0,0);
}

.hovereffect:hover h2:after {
  -webkit-transform: translate3d(0,0,0);
  transform: translate3d(0,0,0);
}

.hovereffect a, .hovereffect p {
  color: #FFF;
  opacity: 0;
  filter: alpha(opacity=0);
  -webkit-transition: opacity 0.35s, -webkit-transform 0.35s;
  transition: opacity 0.35s, transform 0.35s;
  -webkit-transform: translate3d(100%,0,0);
  transform: translate3d(100%,0,0);
}

.hovereffect:hover a, .hovereffect:hover p {
  opacity: 1;
  filter: alpha(opacity=100);
  -webkit-transform: translate3d(0,0,0);
  transform: translate3d(0,0,0);
}

span{
  color: #111;
}
.footer{
  position: fixed;
  left:0;
  bottom: 0;
  width: 100%;
  background-color: #63bc46;
  color:black;
  text-align: center;
}
=======
  /* The sidebar links */
  
  .sidebar a {
	padding: 8px 8px 8px 32px;
	text-decoration: none;
	font-size: 0.7em;
	color: white;
	display: block;
	transition: 0.3s;
  }
  
  /* When you mouse over the navigation links, change their color */
  
  .sidebar a:hover {
	color: #63BC46;
  }
  
  /* Position and style the close button (top right corner) */
  
  .sidebar .closebtn {
	position: absolute;
	top: 0;
	right: 25px;
	font-size: 36px;
	margin-left: 50px;
  }
  
  /* The button used to open the sidebar */
  
  .openbtn {
	font-size: 20px;
	cursor: pointer;
	background-color: #111;
	color: white;
	padding: 10px 15px;
	border: none;
  }
  
  .openbtn:hover {
	background-color: #444;
  }
  
  /* Style page content - use this if you want to push the page content to the right when you open the side navigation */
  
  #main {
	transition: margin-left .5s;
	/* If you want a transition effect */
	padding: 20px;
  }
  
  /* On smaller screens, where height is less than 450px, change the style of the sidenav (less padding and a smaller font size) */
  
  @media screen and (max-height: 450px) {
	.sidebar {
	  padding-top: 15px;
	}
	.sidebar a {
	  font-size: 18px;
	}
  }
  
  .navbar-brand {
	font-family: 'Gowun Dodum', sans-serif;
	color: black;
	font-weight: bolder;
  }
  
  .carousel img {
	width: 100px;
	height: 400px;
  }
  
  .btn a {
	color: black;
  }
>>>>>>> 0b088f20cc06084d783b4f1f097786b51eda73eb
