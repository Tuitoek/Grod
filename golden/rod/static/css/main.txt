@import url('https://fonts.googleapis.com/css2?family=Gowun+Dodum&display=swap');
.container {
  background-color: #63BC46;
}

.cart {
  border-style: none;
  background-color: goldenrod;
  width: 8em;
  height: 3em;
  font-size: 1em;
  color: white;
  padding: 15px 25px;
  text-decoration: none;
}

.cart:hover {
  background-color: #7A489A;
}

/* The sidebar menu */

.sidebar {
  height: 100%;
  /* 100% Full-height */
  width: 0;
  /* 0 width - change this with JavaScript */
  position: fixed;
  /* Stay in place */
  z-index: 1;
  /* Stay on top */
  top: 0;
  left: 0;
  background-color: #111;
  /* Black*/
  overflow-x: hidden;
  /* Disable horizontal scroll */
  padding-top: 60px;
  /* Place content 60px from the top */
  transition: 0.5s;
  /* 0.5 second transition effect to slide in the sidebar */
}

/* The sidebar links */

.sidebar a {
  padding: 8px 8px 8px 32px;
  text-decoration: none;
  font-size: 0.7em;
  color: white;
  display: block;
  transition: 0.3s;
}

/* When you mouse over the navigation links, change their color */

.sidebar a:hover {
  color: #63BC46;
}

/* Position and style the close button (top right corner) */

.sidebar .closebtn {
  position: absolute;
  top: 0;
  right: 25px;
  font-size: 36px;
  margin-left: 50px;
}

/* The button used to open the sidebar */

.openbtn {
  font-size: 20px;
  cursor: pointer;
  background-color: #111;
  color: white;
  padding: 10px 15px;
  border: none;
}

.openbtn:hover {
  background-color: #444;
}

/* Style page content - use this if you want to push the page content to the right when you open the side navigation */

#main {
  transition: margin-left .5s;
  /* If you want a transition effect */
  padding: 20px;
}

/* On smaller screens, where height is less than 450px, change the style of the sidenav (less padding and a smaller font size) */

@media screen and (max-height: 450px) {
  .sidebar {
    padding-top: 15px;
  }
  .sidebar a {
    font-size: 18px;
  }
}

.navbar-brand {
  font-family: 'Gowun Dodum', sans-serif;
  color: black;
  font-weight: bolder;
}

.carousel img {
  width: 100px;
  height: 400px;
}

.btn a {
  color: black;
}

/* home - (_layout.scss)
*
* -------------------------------------------------------------------  */

#home {
  /* width: 100%; */
  height: 70vh;
  background: url(../IMG/hu-nh-ch-ng-JBThDJlIljs-unsplash.jpg)no-repeat center center fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  /* background-size: 100% auto; */
  background-attachment: fixed;
}

/* fOOD DESTR - (_layout.scss)
*
* -------------------------------------------------------------------  */

#distribution {
  /* width: 100%; */
  height: 30vh;
  background-color: blue;
}

#distribution h2 {
  font-style: oblique;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
  font-size: 2rem;
  padding-top: 3rem;
  color: #ffff;
  text-align: center;
}

/* PRODUCT - (_layout.scss)
*
* -------------------------------------------------------------------  */

#product {
  /* width: 100%; */
  height: 100vh;
}
#product .ptitle{
  font-size: 3rem;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
  font-weight: 900;
  margin-left: 3rem;
}

/* ABOUT US - (_layout.scss)
*
* -------------------------------------------------------------------  */

#About{
  height: 100vh;
}
#About .row{
  height: 100vh;
  
}
#About .Atitle{
  font-size: 3rem;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
  font-weight: 900;
  margin-left: 3rem;
}


/* hover========================== */
.hovereffect {
  width: 100%;
  height: 50%;
  float: left;
  overflow: hidden;
  position: relative;
  text-align: center;
  cursor:default;
  background: #a3962e;
}

.hovereffect .overlay {
  width: 100%;
  height: 100%;
  position: absolute;
  overflow: hidden;
  top: 0;
  left: 0;
  padding: 50px 20px;
}




.hovereffect h2 {
  text-transform: uppercase;
  color: #fff;
  text-align: center;
  position: relative;
  font-size: 17px;
  overflow: hidden;
  padding: 0.5em 0;
  background-color: transparent;
}

.hovereffect h2:after {
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%;
  height: 2px;
  background: #fff;
  content: '';
  -webkit-transition: -webkit-transform 0.35s;
  transition: transform 0.35s;
  -webkit-transform: translate3d(-100%,0,0);
  transform: translate3d(-100%,0,0);
}

.hovereffect:hover h2:after {
  -webkit-transform: translate3d(0,0,0);
  transform: translate3d(0,0,0);
}

.hovereffect a, .hovereffect p {
  color: #FFF;
  opacity: 0;
  filter: alpha(opacity=0);
  -webkit-transition: opacity 0.35s, -webkit-transform 0.35s;
  transition: opacity 0.35s, transform 0.35s;
  -webkit-transform: translate3d(100%,0,0);
  transform: translate3d(100%,0,0);
}

.hovereffect:hover a, .hovereffect:hover p {
  opacity: 1;
  filter: alpha(opacity=100);
  -webkit-transform: translate3d(0,0,0);
  transform: translate3d(0,0,0);
}